extensions/amp-fit-text/0.1/amp-fit-text.css

```diff
@@ -17,17 +17,38 @@
 .i-amphtml-fit-text-content,
 .i-amphtml-fit-text-content.i-amphtml-fill-content {
   display: block;
-  /* @alternative */ display: flex;
+  /* @alternative */
+  display: flex;
   flex-direction: column;
   flex-wrap: nowrap;
   justify-content: center;
 }
 
+.i-amphtml-fit-text-content {
```

Comment:
For these styles and others formerly set with `setStyles`, since they had the highest specificity due to being inline, we should replicate that here by giving the same properties `!important`.

Reply:
Was the implicit `!important` actually desired or just a side effect?

Reply:
@samouri How hard would it be to support `setStyles` in the compiler? I would imagine including inline styles in compiled code shouldn't be impossible (it lives in `#core` so the function is dependency-minimal). Is CSS better than inline in some way? Curious why setStyles was used to begin with

Reply:
I would consider the implicit `!important` to be desired whenever we don't want authors overriding styles. In this case zindex should not get smaller and visibility should stay hidden. 

At the same time, it is pretty common to use the `setStyles` helper, so I can't speak to the original intention. But either way we should use it for stability.

Reply:
> @samouri How hard would it be to support setStyles in the compiler? I would imagine including inline styles in compiled code shouldn't be impossible (it lives in #core so the function is dependency-minimal). Is CSS better than inline in some way? Curious why setStyles was used to begin with

Both very interesting questions! For the former....it seems possible to me, but still less good than using CSS. It is almost always better to involve as little JS as possible.

re. why setStyles was used to begin with...I can't say

Reply:
For something like the line height it seems useful to be able to have it defined in one place. It's interesting that in the context of the compiler, the more that lives in the JS the more that can be pre-compiled, but worse for uncompiled pages/elements 🤔

Reply:
> How hard would it be to support setStyles in the compiler?

In its current incarnation, it won't even work in JS-only mode since it depends on vendor detection for special vendor prefixing

Reply:
Nice data/benchmarking!